pyvrp/tests/test_Individual.py

```diff
@@ -262,8 +262,7 @@ def test_str_contains_essential_information():
 
         # Last line should contain the cost
         # TODO what do we want with cost in str representation?
-        # assert_(
-        #     str(individual.cost(cost_evaluator)) in str_representation[-1])
+        # assert_(str(cost_evaluator(individual)) in str_representation[-1])
```

Comment:
Maybe we can check that the distance is in the representation? Then we can add other attributes later, as wanted/needed.

Reply:
Done